Reference/C.9.2.bpmn

```diff
@@ -0,0 +1,3882 @@
+                                     sourcelang=""
+                                     novalue="0"
+                                     lang-independent="v0">
+                     <adonis:value>Per year</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:attribute name="COLLECTION_DATATYPE_OUTPUT"
+                                     type="BOOL"
+                                     notebook="1"
+                                     sourcelang=""
+                                     novalue="0">
+                     <adonis:value>false</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:attribute name="A_QUANTITY"
+                                     type="INTEGER"
+                                     notebook="1"
+                                     sourcelang=""
+                                     novalue="0">
+                     <adonis:value>0</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:attribute name="AUDITING"
+                                     type="BOOL"
+                                     notebook="1"
+                                     sourcelang=""
+                                     novalue="0">
+                     <adonis:value>false</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:attribute name="OBJECT_TYPE"
+                                     type="ENUM"
+                                     notebook="1"
+                                     sourcelang=""
+                                     novalue="0"
+                                     lang-independent="v2">
+                     <adonis:value>No entry</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:attribute name="DISPLAY_NAME"
+                                     type="BOOL"
+                                     notebook="1"
+                                     sourcelang=""
+                                     novalue="0">
+                     <adonis:value>true</adonis:value>
+                     <adonis:metavalue/>
+                  </adonis:attribute>
+                  <adonis:record name="A_ATTACHMENT_LIST_CS"/>
+                  <adonis:record name="OPEN_QUESTIONS"/>
+               </adonis:instance>
+            </extensionElements>
+            <outgoing>_1a7ce630-b380-465e-ba3c-9b205f85f0ad</outgoing>
+            <messageEventDefinition id="_1ccfd585-590c-4abf-84b3-143650b8c0a0"/>
+         </startEvent>
+         <callActivity name="Document Request"
+                        id="_1c9f4f0d-d193-4643-8201-0843a9a937fe"
+                        isForCompensation="false"
+                        calledElement="process_1c9f4f0d-d193-4643-8201-0843a9a937fe"
```

Comment:
The called element here is not defined or imported, there is no way to know if the + marker should be displayed in the diagram.